packages/pluggableWidgets/column-chart-native/src/utils/__tests__/StyleUtils.spec.ts

```diff
@@ -0,0 +1,212 @@
+import { ColumnChartAxisStyle, ColumnChartGridStyle, ColumnChartStyle } from "../../ui/Styles";
+
+describe("StyleMappers", () => {
+    describe("mapToGridStyle", () => {
+        it("maps the relevant styles", () => {
+            const gridStyle: ColumnChartGridStyle = {
+                backgroundColor: "orange",
+                dashArray: "5,5",
+                lineColor: "transparent",
+                width: 10,
+                padding: 1,
+                paddingHorizontal: 2,
+                paddingVertical: 3,
+                paddingBottom: 30,
+                paddingLeft: 30,
+                paddingRight: 10,
+                paddingTop: 10
+            };
+
+            const expectedResult = {
+                background: {
+                    fill: gridStyle?.backgroundColor
+                }
+            };
+
+            expect(mapToGridStyle(gridStyle)).toStrictEqual(expectedResult);
+        });
+    });
+
+    describe("mapToAxisStyle", () => {
+        it("maps the relevant styles", () => {
+            const gridStyle: ColumnChartGridStyle = {
+                backgroundColor: "orange",
+                dashArray: "5,5",
+                lineColor: "transparent",
+                width: 10,
+                padding: 1,
+                paddingHorizontal: 2,
+                paddingVertical: 3,
+                paddingBottom: 30,
+                paddingLeft: 30,
+                paddingRight: 10,
+                paddingTop: 10
+            };
+
+            const axisStyle: ColumnChartAxisStyle<"Y"> = {
+                color: "green",
+                dashArray: "10,10",
+                fontFamily: "Times New Roman",
+                fontSize: 20,
+                fontStyle: "italic",
+                fontWeight: "100",
+                lineColor: "purple",
+                width: 5,
+                label: {
+                    relativePositionGrid: "left",
+                    alignSelf: "center"
+                }
+            };
+
+            const expectedResult = {
+                axis: {
+                    stroke: axisStyle?.lineColor,
+                    strokeDasharray: axisStyle?.dashArray,
+                    strokeWidth: axisStyle?.width
+                },
+                grid: {
+                    stroke: gridStyle?.lineColor,
+                    strokeDasharray: gridStyle?.dashArray,
+                    strokeWidth: gridStyle?.width
+                },
+                tickLabels: {
+                    fill: axisStyle?.color,
+                    fontFamily: axisStyle?.fontFamily,
+                    fontSize: axisStyle?.fontSize,
+                    fontStyle: axisStyle?.fontStyle,
+                    fontWeight: axisStyle?.fontWeight
+                }
+            };
+
+            expect(mapToAxisStyle(gridStyle, axisStyle)).toStrictEqual(expectedResult);
+        });
+    });
+
+    describe("mapToBarStyles", () => {
```

Comment:
This is named incorrect although its not an issue as this code will be refactored.